src/lbaf/IO/configurationValidator.py

```diff
@@ -27,61 +33,99 @@ def __init__(self, config_to_validate: dict, logger: Logger):
         self.__config_to_validate = config_to_validate
         self.__skeleton = Schema({
             Or("from_data", "from_samplers", only_one=True): dict,
-            "work_model": {"name": And(str, lambda c: c in ALLOWED_WORK_MODELS,
-                                       error=f"{get_error_message(ALLOWED_WORK_MODELS)} needs to be chosen"),
-                           "parameters": {"alpha": float,
-                                          "beta": float,
-                                          "gamma": float}},
+            "work_model": {
```

Comment:
formatting was readable, no need to add 60 extra lines to the file

Reply:
Extra lines are good and generally a way to enhance one's appreciation of life.